.github/workflows/tests.yml

```diff
@@ -217,10 +235,23 @@ jobs:
           mkdir %UserProfile%\cbtmp
           for /d %%F in (%UserProfile%\cbtmp_serial\*) do rd /s /q "%%F"
           for /d %%F in (%UserProfile%\cbtmp\*) do rd /s /q "%%F"
-          call conda create -n blarg -yq --download-only python=2.7||exit 1
-          call conda create -n blarg -yq --download-only python=3.8||exit 1
+          call conda create -n blarg -yq --download-only python=3.9||exit 1
+          call conda create -n blarg -yq --download-only python=3.10||exit 1
           call conda create -n blarg -yq --download-only python cmake||exit 1
           mkdir ${{ github.workspace }}\\pytest-replay
+
+          echo "Installing Support Libraries"
+          pip install -e .
+          pip install setuptools wheel twine
+          pip install pytest-html
+          pip install allure-pytest
```

Comment:
probably allure-pytest is the only one that is required and not already installed?

Reply:
yes that is true.. unless we make another section int he job to do installation and set up before..